### DISCUSSION TITLE
Audio delay in VMs

### DISCUSSION THREAD

**Discussion by daxhuiberts** [utmapp/UTM#3707] [UTM Help (macOS)]
Hi, I have a MacBook Pro M1 and I am amazed how well UTM works on this machine.

I am able to run Linux (Fedora), MacOS and Windows flawlessly with the spice guest tools.

One oddity I do have is that audio is delayed a bit in all VMs I tried (win/mac/linux). e.g. when I play a youtube music video and I play/pause the video, the audio is always lagging behind about 200/300 ms. It continues playing a slight bit longer after pausing and only starts playing a bit after continueing.

Considering I have the problem in all three VMs (win/mac/linux) it sounds (haha) like it doesn't have anything to do with the VM, but with the host. I can't seem to figure out what might cause the problem or how to debug it further. I can't find any similar issues in the issue tracker and discussion section here either. Any ideas what might cause this?

**Comment by RDMurray** [comment]
Hi, did you ever find a solution?
